lib/tasks/deliver.rake

```diff
@@ -0,0 +1,6 @@
+namespace :deliver do
+  task :to_subscriber, [:id] => :environment do |_t, args|
+    subscriber = Subscriber.find(args[:id])
+    DeliverToSubscriber.call(subscriber, Services.email_sender)
```

Comment:
I think this will need to be given as keyword arguments, `call(subscriber: subscriber, email_sender: Services.email_sender)`